api/team.go

```diff
@@ -121,6 +121,11 @@ func inviteMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if !c.App.SessionHasPermissionToTeam(c.Session, c.TeamId, model.PERMISSION_ADD_USER_TO_TEAM) {
+		c.SetPermissionError(model.PERMISSION_INVITE_USER)
```

Comment:
In the event a user has `PERMISSION_INVITE_USER` but is missing `PERMISSION_ADD_USER_TO_TEAM`, is there a way we can distinguish that failure in the logs/response? I could foresee some difficulty down the road where a client might want to detangle these two cases.

Reply:
I was using the "mimic" approach. from the api4 implementation.